variant/tokenizers/protein_delins.py

```diff
@@ -1,20 +1,27 @@
-#\\b{}\\d+(_{}\\d+)?((DEL|del)|(INS|ins)|>)+{}*\\b
-#protein digits (protein digits)? (del|ins|>) protein
-#why "one or more" on the del/ins/>
+"""Module for Protein DelIns Tokenizer."""
 import re
-
 from typing import Pattern, Optional
-
 from .tokenizer import Tokenizer
 from .caches import AminoAcidCache
-from ..models import Token
+from variant.schemas.token_response_schema import Token, TokenMatchType
+
+"""
+\\b{}\\d+(_{}\\d+)?((DEL|del)|(INS|ins)|>)+{}*\\b
+protein digits (protein digits)? (del|ins|>) protein
+why "one or more" on the del/ins/>
+"""
```

Comment:
I'm not entirely sure. I think @acoffman wrote this. We can probably delete this though. It doesn't look like the regex is correct since Protein Deletion-Insertion should have `delins`.